_posts/2016-02-08-terminal-commands.md

```diff
@@ -0,0 +1,233 @@
+# Importance of using the terminal?
+
+Under multiple platforms the Operating System has the GUIs (graphical user interfaces), where you can as a user point, click and drag on multiple components of the OS, which makes the user experience extremely easy and friendly. 
```

Comment:
@marticam  I'd probably remove the "the" before "GUIs". I would even rephrase this to read:

"Nowadays, most Operating Systems (OS) include a Graphical User Interface (GUI), where users can point, click and drag on multiple components of the GUI, which makes it much easier for users to interact with the OS."

Just an idea, you don't have to write it that way though. @ssjuampi23 what do you think?